app/flatpak-tty-utils.c

```diff
@@ -522,14 +522,37 @@ flatpak_print_escaped_string (const char        *s,
   g_print ("%s", escaped);
 }
 
+static gboolean
+use_progress_escape_sequence (void)
+{
+  static gsize pty_progress_once = 0;
+  enum {
+    PTY_PROGRESS_ENABLED = 1,
+    PTY_PROGRESS_DISABLED = 2
+  };
+
+  if (g_once_init_enter (&pty_progress_once))
+    {
+      // FIXME: make this opt-out for Flatpak 1.18
+      if (g_strcmp0 (g_getenv ("FLATPAK_PTY_PROGRESS"), "1") == 0)
```

Comment:
It's a terminal (tty), not necessarily a pseudo-terminal (/dev/pts), so this should be `FLATPAK_TTY_PROGRESS` or `FLATPAK_TERMINAL_PROGRESS` or something similar.

Reply:
Thanks. Opted for `FLATPAK_TTY_PROGRESS`.